pkg/lib/serverTime.js

```diff
@@ -562,26 +563,19 @@ function ChangeSystimeBody({ state, errors, change }) {
     }
 
     const ntp_servers = (
-        <table>
-            <tbody>
-                { custom_ntp.servers.map((s, i) => (
-                    <tr key={i}>
-                        <td style={{ width: "100%" }}>
-                            <input type="text" className="form-control" value={s} placeholder={_("NTP server")}
-                onChange={event => change_server(event, i, event.target.value)} />
-                        </td>
-                        <td>
-                            <Button variant="secondary" onClick={event => add_server(event, i)} icon={ <PlusIcon /> } />
-                        </td>
-                        <td>
-                            <Button variant="secondary" onClick={event => remove_server(event, i)}
-                                    icon={ <CloseIcon /> }
-                                    isDisabled={i == custom_ntp.servers.length - 1} />
-                        </td>
-                    </tr>))
-                }
-            </tbody>
-        </table>);
+        custom_ntp.servers.map((s, i) => (
+            <Flex className="ntp-server-input-group" spaceItems={{ default: 'spaceItemsSm' }} key={i}>
+                <FlexItem grow={{ default: 'grow' }}>
+                    <TextInput value={s} placeholder={_("NTP server")} aria-label={_("NTP server")}
```

Comment:
placeholders are used wrongly here. (and in many other places in cockpit) 
placeholders are supposed to contain examples of the input, not description of the input. Since it existed here, it's not a regression.

Reply:
I agree with this! Thanks for pointing it out. :+1: :100: 

 I talked about this specific issue about the placeholder with @jelly in the call we had (and I think in the issue too). The idea is to handle this problem in a follow-up that redoes the UI anyway. We wanted to get in a fix for it not being PF first (as it's in a screenshot in the release notes and looks _really bad_ otherwise), then fix it properly after.